tests/components/unifi/conftest.py

```diff
@@ -234,6 +244,12 @@ def fixture_port_forward_data() -> list[dict[str, Any]]:
     return []
 
 
+@pytest.fixture(name="traffic_rule_payload")
+def traffic_rule_payload_data() -> list[dict[str, Any]]:
+    """Traffic rule data."""
+    return []
+
+
```

Comment:
```suggestion
@pytest.fixture(name="traffic_rule_payload")
def fixture_traffic_rule_data() -> list[dict[str, Any]]:
    """Traffic rule data."""
    return []


```
Place it between `fixture_system_information_data` and `fixture_wlan_data`